test/test_ops.py

```diff
@@ -127,6 +135,11 @@ def reduction_dtype_filter(op):
 
 aten = torch.ops.aten
 
+def any_common_cpu_device_one():
+    return OpDTypes.any_common_cpu_xpu_one if TEST_XPU else OpDTypes.any_common_cpu_cuda_one
+
+def has_gpu_device(devices: List[str]):
+    return "cuda" in devices or "xpu" in devices
```

Comment:
The two functions are common enough. Should them be part of `common_utils.py`?

Reply:
Done